app.py

```diff
@@ -28,6 +28,21 @@ def home():
     if request.method == "POST":
         code = request.form.get('getCode')
         code = code.upper()
+        #Extratcing name of course for wiki links
+        Course_info=code
+        #To extract Name of Course
+        Course_info=Course_info.split(":")[1][1:]
+        CourseName=Course_info
+        #Forming a string which can be used as a link
+        course_info_wiki_link=""
+        for i in range(0,len(Course_info)):
+            if(i==0):
+                course_info_wiki_link=course_info_wiki_link+"_"+Course_info[i]
```

Comment:
Must have a space before and after the assignment operator or any other operator.